cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/DiskSpacePricingDTO.java

```diff
@@ -1,8 +1,24 @@
 package com.cloudesire.platform.apiclient.dto.model.dto;
 
 import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+import java.math.BigDecimal;
+import java.util.Map;
 
 @ApiModel( description = "Defines pricing of disk usage of a VM" )
 public class DiskSpacePricingDTO extends CloudPricingDTO
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS2160)